rancher/v1.2/en/cattle/adding-load-balancers/index.md

```diff
@@ -285,206 +322,201 @@ Rancher supports wildcards when adding host based routing. The following wildcar
-    io.rancher.loadbalancer.target.service1: dummy1:81=81
-    io.rancher.loadbalancer.target.service2: 81=81
-    # Add in dummy rule for service2 to force the traffic coming to 80 to not be redirected to service1
-    io.rancher.loadbalancer.target.service2: dummy2:80=80
-  tty: true
-  image: rancher/load-balancer-service
-  links:
-  - service1:service1
-  - service2:service2
-  stdin_open: true
+version: '2'
+services:
+  lb:
+    scale: 1
+    lb_config:
+      certs:
+      - <certName>
+      default_cert: <defaultCertName>
 ```
 
-#### Internal Load Balancer
+#### Custom configuration
 
-To set an internal load balancer, the listening ports are listed under `expose` instead of `ports`.
+For advanced users, you can specify custom configuration to the load balancer in the `rancher-compose.yml`. Please refer to the [HAProxy documentation](http://cbonte.github.io/haproxy-dconv/configuration-1.5.html) for details on the available options you can add for the Rancher's HAproxy load balancer.
 
-##### Example `docker-compose.yml`
+##### Example `rancher-compose.yml`
 
 ```yaml
-lb:
-  image: rancher/load-balancer-service
-  # Instead of using ports, we use expose to define that it will be private ports
-  expose:
-  # Listen on private port 80 and direct traffic to private port 80 of the service
-  - 80:80
-  # Listen on private port 82 and by default forward traffic to private port 81 using HTTP
-  - 82:81
-  links:
-  # Any service that is a target will be listed as a link
-  - web1:web1
+version: '2'
+services:
+  lb:
+    scale: 1
+    lb_config:
+      config: <CustomConfig>
+  health_check:
+    port: 42
+    interval: 2000
+    unhealthy_threshold: 3
+    healthy_threshold: 2
+    response_timeout: 2000
 ```
 
-<br>
+#### Stickiness Policy
 
-##### Example  `rancher-compose.yml`
+If you want to specify stickiness policy, you can update the policies in `rancher-compose.yml`.
+
+##### Example `rancher-compose.yml`
 
 ```yaml
-lb:
-  # Two load balancer containers
-  scale: 2
-  load_balancer_config:
-    name: lb config
+version: '2'
+services:
+  lb:
+    scale: 1
+    lb_config:
+      stickiness_policy:
+        name: <policyName>
+        cookie: <cookieInfo>
+        domain: <domainName>
+        indirect: false
+        nocache: false
+        postonly: false
+        mode: <mode>
   health_check:
     port: 42
     interval: 2000
     unhealthy_threshold: 3
     healthy_threshold: 2
     response_timeout: 2000
-web1:
-  scale: 1
 ```
 
-<br>
+### Rancher Compose Examples
 
-#### SSL Termination
-
-Rancher Compose uses labels to define the SSL ports to be used in SSL termination. The [certificates]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/environments/certificates/) must be added into Rancher and are defined in the `rancher-compose.yml`.
+#### Load Balancer Example (L7)
 
 ##### Example `docker-compose.yml`
 
 ```yaml
-lb:
+version: '2'
+services:
+  web:
+    image: nginx
+  lb:
+    image: rancher/lb-service-haproxy
```

Comment:
thinking about this one...we are going to tag the haproxy image, and won't be using "latest" tag after all. Current tag is v0.4.1, but it might change towards GA. @joshwget need your opinion on that - are we making any validation against the image officially supported by cattle? "lb.instance.image" global setting. So user can put whatever he wants, but we 1) document the officially supported haproxy tags for every rancher release @deniseschannon 2) ranhcer-compose will do the check and present a warning if user specifies non-supported version